sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/CHANGELOG.md

```diff
@@ -10,6 +10,11 @@
 
 ### Other Changes
 
+## 1.0.1 (2024-03-29)
+
+### Other changes
+- Added a direct reference to `Newtonsoft.Json` v13.0.3 to hoist up the version used by `Apache.Avro`, which has known security vulnerabilities.
```

Comment:
I don't see the reference in the csproj - is it somewhere else?

Reply:
Yeah, it was intentionally made only on a hotfix branch, as it's not something that we intend to bring forward.   The reference in `Apache.Avro` has already been bumped, we're just waiting on a new release.   We don't expect to release this library again without bumping `Apache.Avro`